filters/redirect/redirect_test.go

```diff
@@ -24,3 +25,26 @@ func TestRedirect(t *testing.T) {
 		t.Error("invalid location")
 	}
 }
+
+func TestRedirectRelative(t *testing.T) {
+	spec := &Redirect{}
+	f, err := spec.MakeFilter("redirect0", []interface{}{float64(302), "/relative/url"})
```

Comment:
float64: most forgiving number type. checked earlier, it doesn't cause issues during int conversions. Also, see: http://localhost:8989/pkg/encoding/json/#Unmarshal